### ISSUE TITLE
Sorting option in plugin store menu

### ISSUE THREAD

**Issue by DRYN07** [Flow-Launcher/Flow.Launcher#2845]
It is quite hard to follow new/updated plugins if don't check regularly this page. It would be very helpful to sort plugins by:
- name
- release date
- updated date

**Comment by tupaschoal** [comment]
This would indeed be very useful. Just installed the app today and it was a bit painful going through the list of plugins, installing one to then see flow refresh and bring it up again with a different sorting.
